tests/Unit/GeneralAssemblyTest.php

```diff
@@ -166,4 +167,28 @@ public function test_passive_users_get_excused_automatically(): void
         $this->assertTrue($excused->contains($userPassive1) && $excused->contains($userPassive2));
         $this->assertNotNull($excused->first()->pivot->comment); // Check if excuse reason is set
     }
+
+    /**
+     * @return void
+     */
+    public function test_new_students_pass_requirements(): void
+    {
+        $user = User::factory()->create(['verified' => true]);
+        $user->educationalInformation()->save(EducationalInformation::factory()->make());
+        $user->educationalInformation->update(['year_of_acceptance' => now()->year]);
+
+        $generalAssembly = GeneralAssembly::factory()->create();
```

Comment:
```suggestion
        $generalAssembly = GeneralAssembly::factory()->create(['closed_at' => Carbon::createFromDate(now()->year, 2, 15);
```
and delete the update. Do the same for the second assembly.